src/overlays/actors/ovl_En_Ru1/z_en_ru1.h

```diff
@@ -4,9 +4,57 @@
 #include <ultra64.h>
 #include <global.h>
 
+#include <overlays/actors/ovl_Bg_Bdan_Objects/z_bg_bdan_objects.h>
+#include <overlays/actors/ovl_Door_Warp1/z_door_warp1.h>
+
 typedef struct {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x250];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ Vec3s unk_190;
+    /* 0x0196 */ char unk_196[0x60];
+    /* 0x01F6 */ Vec3s unk_1F6;
+    /* 0x01FC */ char unk_1FC[0x60];
```

Comment:
I wont have time to review this whole thing tonight but I wanted to write this down now while Im thinking about it. actorDrawTable is a table of rotation values used by skelanime for animations. transitionDrawTable is used for transitioning between animations. The size of the array is the number of limbs. You can get the limb count easily by lookinf at SkelAnime_Init(SV). Also, I would change the limb count to decimal in that function call and anywhere else it may be used.
```suggestion
    /* 0x0190 */ Vec3s actorDrawTable[17];
    /* 0x01F6 */ Vec3s transitionDrawTable[17];
```

Reply:
ahhh okay. Thanks, that's very enlightinging. I believe this can be fixed in ru2 as well. I'll do that tomorrow.

Reply:
This may be a better conversation for discord, but I feel like the name actorDrawTable is not really appropriate then. I know that's beyond the scope of this PR but yeah, just thinking about it aloud.. the "actor" part seems redundant, and it's not in the other name. why not "animationDrawTable"?